### PR TITLE
Prepare to release spring related modules

### PR DESCRIPTION
**Pull request by chenrujun**
Prepare to release spring related modules

### CODE REVIEW

**Comment by chenrujun** [comment]
I created an issue about `keyvault-ci` pipeline failure: https://github.com/Azure/azure-sdk-for-java/issues/21677

**Review by yiliuTo — APPROVED** [review]
LGTM

File: sdk/boms/azure-spring-cloud-dependencies/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Release History
 
+
+## 2.5.0 (2021-05-20)
+
```
Comment (yiliuTo) [inline sdk/boms/azure-spring-cloud-dependencies/CHANGELOG.md:5]: I think we could make the changelog of azure-spring-cloud-dependencies aligned with that of other azure spring cloud libraries.

Comment (yiliuTo) [reply sdk/boms/azure-spring-cloud-dependencies/CHANGELOG.md:5]: For some unknown legancy reasons, there is a changelog of azure spring cloud bom, but not of spring boot, perhaps we could consider in the future that we should delete this or add changelog for spring boot as well?

Comment (chenrujun) [reply sdk/boms/azure-spring-cloud-dependencies/CHANGELOG.md:5]: Added.

**Review by yiliuTo — APPROVED** [review]
LGTM

**Pull request merged**